TypeCobol/Compiler/Scanner/ScannerUtils.cs

```diff
@@ -55,4 +55,22 @@ public static class ScannerUtils
 
         return false;
     }
+
+    /// <summary>
+    /// Get the CobolFormatArea in which the given index is into.
+    /// </summary>
+    /// <param name="index">The index to found the corresponding CobolFormatAreas</param>
+    /// <returns> the CobolFormatArea</returns>
+    public static CobolFormatAreas GetAreaFromIndex(int index)
```

Comment:
I think this should return `TextAreaType` instead of `CobolFormatAreas`. In fact `CobolFormatAreas` is not really an enmu but more of a collection of const int. Also the test for `Indicator` is incorrect, it should be

```csharp
        if (index < (int)CobolFormatAreas.Begin_A)
            return CobolFormatAreas.Indicator;
```